src/App.js

```diff
@@ -17,8 +18,36 @@ const tabs = [
   },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+class App extends React.Component {
+  state = {
+    index: 0,
+  }
+
+  onTabSelected = (event) => {
+    const chosenTab = tabs.find(tab => (
+      tab.title === event.target.innerText
+    ));
+
+    this.setState({
+      index: tabs.indexOf(chosenTab),
+    });
```

Comment:
you can also pass tab.id to this function, so you will only need to save it to the state (no need to use find and indexOf) ;)